### PR TITLE
Create book suggest

### PR DESCRIPTION
**Pull request by FVolonnino**
https://trello.com/c/91PtkOCN/27-crear-ruta-para-creaci%C3%B3n-de-un-booksuggestion

### CODE REVIEW

File: app/controllers/api/v1/book_suggestion_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
+      skip_before_action :verify_authenticity_token
```
Comment (Gmarn) [inline app/controllers/api/v1/book_suggestion_controller.rb]: 👏

File: app/controllers/api/v1/book_suggestion_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          render json: 'Book suggestion created and saved!'
+        else
+          render json: 'The book suggestion could not be created'
+        end
+      end
+
+      private
+
+      def create_params
+        params.permit(:title, :author, :publisher, :link, :year,
```
Comment (Gmarn) [inline app/controllers/api/v1/book_suggestion_controller.rb]: 👏

File: config/routes.rb [hunk 3]
```diff
@@ -3,7 +3,8 @@
   ActiveAdmin.routes(self)
   api_version(:module => "Api/V1", :path => {:value => "api/v1"}) do
     resources :book, only: [:index, :show]
-    resources :rent, only: [:index, :create]
+    resources :rent, only: [:index, :create, :show]
```
Comment (Gmarn) [inline config/routes.rb]: You forgot to remove this `:show`

File: config/routes.rb [hunk 4]
```diff
@@ -3,7 +3,8 @@
...
+    resources :book_suggestion, only: [:create]
```
Comment (Gmarn) [inline config/routes.rb]: 👏

File: app/controllers/api/v1/book_suggestion_controller.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+module Api
+  module V1
+    class BookSuggestionController < ApplicationController
+      skip_before_action :verify_authenticity_token
+
+      def create
+        bs = BookSuggestion.new(create_params)
```
Comment (Gmarn) [inline app/controllers/api/v1/book_suggestion_controller.rb]: Better to use more explicit name.

File: app/controllers/api/v1/book_suggestion_controller.rb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        bs.user_id = current_user ? current_user.id : nil
```
Comment (Gmarn) [inline app/controllers/api/v1/book_suggestion_controller.rb]:
```suggestion
        bs.user = current_user ? current_user : nil
```

Prettier to not use the id ;)

Comment (FVolonnino) [reply app/controllers/api/v1/book_suggestion_controller.rb]: Oh, I didn't you could do that. Nice.

**Pull request merged**